Build/create-sbom.ps1

```diff
@@ -0,0 +1,97 @@
+﻿param(
+    [string] $TempFolder = "Build\BuildOutput\temp\sbom",
+    [Parameter(mandatory=$true)] [string] $NpmProjectJson,
```

Comment:
hardcode it, based on the externally supplied working directory. Working Directory defaults to THIS.